src/components/trends/common/TrendsSVGContainer.js

```diff
@@ -1,3 +1,20 @@
+const BUMPERS = {
```

Comment:
These constants should probably be moved below the import statements

Reply:
Oh yeah they should.